sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -98,7 +98,8 @@
 
 /**
  * An unbounded source and a sink for <a href="http://kafka.apache.org/">Kafka</a> topics.
- * Kafka version 0.9 and above are supported.
+ * Kafka version 0.9 and above are supported. It's supported to replace with Kafka client 0.10
```

Comment:
Its not clear what this implies to user. "It's supported to replace with Kafka client for external authentication.". We can remove it or rephrase it.

This PR makes ensures 'Kafka version 0.9 and above are supported.'. It was not accurate earlier.